.github/workflows/main.yaml

```
@@ -8,11 +8,11 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [24.x]
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
```

Zizmor will also suggest...
```suggestion
    - uses: actions/checkout@v4
       with:
         persist-credentials: false
```
See https://docs.zizmor.sh/audits/#artipacked